readme.md

```diff
@@ -1,7 +1,7 @@
 # Product cards
-Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+Replace `<Andrii-OttO>` with your Github username and copy the links to Pull Request description:
+- [DEMO LINK](https://Andrii-OttO.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://Andrii-OttO.github.io/layout_product-cards/report/html_report/)
```

Comment:
Please add these links to Pull Request description

Reply:
Still not added
<img width="955" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/a65ded8d-1564-4bdf-b675-a511c444774d">

Reply:
Consider adding these links to Pull Request description